tests/tests_integration/test_api/test_assets.py

```diff
@@ -114,7 +114,7 @@ def is_integration_test() -> Filter:
 class TestAssetsAPI:
     def test_get(self, cognite_client):
         res = cognite_client.assets.list(limit=1)
-        assert res[0] == cognite_client.assets.retrieve(res[0].id)
+        assert res[0].id == cognite_client.assets.retrieve(res[0].id).id
```

Comment:
I still don't understand what/how this test fails; is it perhaps xdist related? 🤔